package.json

```diff
@@ -33,6 +33,7 @@
     "@monorepolint/config": "0.5.0-alpha.132",
     "@monorepolint/core": "0.5.0-alpha.132",
     "@monorepolint/rules": "0.5.0-alpha.132",
+    "@phenomnomnominal/tsquery": "6.1.3",
```

Comment:
`ts-morph` would be the alternative library here if you decide you don't like `tsquery`, I don't have a strong opinion here